src/test/resources/fixtures/hmh-ri/caliperEventAssessmentStarted.json

```diff
@@ -0,0 +1,45 @@
+{
+   "@context":"http://purl.imsglobal.org/ctx/caliper/v1p1",
+   "type":"AssessmentEvent",
+   "id":"822fe30b-e42f-4287-805e-98ff5b9e9dea",
+   "actor":{
+      "id":"https://www.hmhco.com/ed/users/0f4fedbe-2227-415f-8553-40731a627171",
+      "type":"Person"
+   },
+   "action":"Started",
+   "object":{
+      "id":"61fe17b8a79f4e50b20c2a58f9fa3097",
+      "type":"Assessment",
+      "name":"HMH-RI"
+   },
+   "eventTime":"2016-11-15T10:15:00.000Z",
+   "edApp":{
+      "id":"https://www.hmhco.com/ed",
+      "type":"SoftwareApplication",
+      "version":"v2"
+   },
+   "membership":{
+      "id":"https://www.hmhco.com/ed/users/0f4fedbe-2227-415f-8553-40731a627171",
+      "member": {
+          "id": "https://www.hmhco.com/ed/users/0f4fedbe-2227-415f-8553-40731a627171",
+          "type": "Person"
+      },
+      "type":"Membership",
+      "organization":{
+         "id":"https://www.hmhco.com/ed/organizations/fbb79fa9-ce24-4979-beba-66c57395b0a1",
+         "type":"Organization",
+         "subOrganizationOf":{
+            "id":"https://www.hmhco.com/ed/organizations/7f7b4763-8b91-45ce-b7a4-e051fd79953c",
+            "type":"Organization"
+         }
+      },
+      "roles":[
+         "Learner"
+      ]
+   },
+   "extensions": {
```

Comment:
I'm not sure if these specific extensions should reside out at the same level as Actor and the other core entities etc.... should they be under some inner caliper entity such as target or object maybe....?

Does having them out at the same level as Actor and the other core Caliper entities mean you lose context on what they are specifically associated with?